docs/create_dimensions.md

```diff
@@ -3,7 +3,7 @@
 In Maximo Asset Monitor, dimensions are a way of aggregating your sensor data in different hierarchical categories and locations. Using a REST API call, you are able to push JSON into the Monitor data lake to assign dimensions to different sensor device IDs.  
```

Comment:
We should say "These are 2 examples. using categories and locations."  There are others.   Again remove heirarchy.